packages/ui/src/components/icons/IconSolidGithub.tsx

```diff
@@ -13,7 +13,7 @@ export const IconSolidGithub = ({ size = '1em', ...props }: IconProps) => {
 			{...props}
 		>
 			<path
-				fill="#6F6E77"
+				fill="currentColor"
```

Comment:
Added this so that the GitHub icon is disabled when the button is disabled

Reply:
nice